docs/en/guides/81-how-databend-works/03-how-databend-json-variant-works.md

```diff
@@ -0,0 +1,314 @@
+│                 JSON Access Syntax Options                      │
+├─────────────────────────────────────────────────────────────────┤
+│                                                                 │
+│  Original JSON:                                                 │
+│  {                                                              │
+│    "user": {                                                    │
+│      "profile": {                                               │
+│        "name": "Alice",                                         │
+│        "settings": {                                            │
+│          "theme": "dark"                                        │
+│        }                                                        │
+│      }                                                          │
+│    }                                                            │
+│  }                                                              │
+│                                                                 │
+│  Access Options:                                                │
+│  ┌──────────────────────────────────────────────────────┐       │
+│  │ 1. Square Bracket Notation:                          │       │
+│  │    data['user']['profile']['settings']['theme']      │       │
+│  │                                                      │       │
+│  │ 2. Colon Notation:                                   │       │
+│  │    data:user:profile:settings:theme                  │       │
+│  │                                                      │       │
+│  │ 3. Mixed Notation with Dots:                         │       │
+│  │    data['user']['profile'].settings.theme            │       │
+│  │    data:user:profile.settings.theme                  │       │
+│  └──────────────────────────────────────────────────────┘       │
+│                                                                 │
+└─────────────────────────────────────────────────────────────────┘
+```
+
+For more details on access syntax, see the [Variant documentation](https://docs.databend.com/sql/sql-reference/data-types/variant#accessing-elements-in-json).
+
+### Rich Function Support
+
+Databend provides a comprehensive set of functions for working with JSON data, organized by their purpose:
+
+```
+┌─────────────────────────────────────────────────────────────────┐
+│                 JSON Function Categories                        │
+├─────────────────────────────────────────────────────────────────┤
+│                                                                 │
+│  1. Basic Operations:                                           │
+│     • Parsing and Validation:                                   │
+│       - PARSE_JSON, CHECK_JSON                                  │
+│     • Object Access and Extraction:                             │
+│       - GET, GET_PATH, GET_IGNORE_CASE, OBJECT_KEYS             │
```

Comment:
we also support PostgreSQL syntax json operator https://docs.databend.com/sql/sql-commands/query-operators/json